tools/experiments/experiments.js

```diff
@@ -101,6 +101,7 @@ const EXPERIMENTS = [
     id: 'no-auth-in-prerender',
     name: 'Delay amp-access auth request until doc becomes visible.',
     spec: '',
+    removalIssue: 'https://github.com/ampproject/amphtml/issues/3824',
```

Comment:
done. added issues and assert.

unfortunately the assert really isnt that helpful since this code isnt part of the runtime and is just accessible on the exerperiments page w/c we normally don't really open.